architecture/infrastructure_components/kubernetes_infrastructure.adoc

```diff
@@ -97,9 +99,51 @@ kubelet:
 - The kubelet listening for HTTP and responding to a simple API to submit a new
  manifest.
 
-[[service-proxy]]
-*Service Proxy*
+=== Service Proxy [[service-proxy]]
 
 Each node also runs a simple network proxy that reflects the services defined in
 the API on that node. This allows the node to do simple TCP and UDP stream
 forwarding across a set of back ends.
+
+=== Node Object Specification
+
+The following is an example specification for a node object in Kubernetes:
+
+====
+
+[source,yaml]
+----
+apiVersion: v1 <1>
+kind: Node <2>
+metadata:
+  creationTimestamp: null
+  labels:
+    kubernetes.io/hostname: node1.example.com <3>
+  name: node1.example.com <4>
+spec:
+  externalID: node1.example.com <5>
+status:
+  nodeInfo:
+    bootID: ""
+    containerRuntimeVersion: ""
+    kernelVersion: ""
+    kubeProxyVersion: ""
+    kubeletVersion: ""
+    machineID: ""
+    osImage: ""
+    systemUUID: ""
+----
+
+<1> *`apiVersion`* defines the API version to use.
+<2> *`kind`* set to `Node` identifies this as a specification for a node
+object.
+<3> *`labels`* lists any labels that have been added to the node.
+<4> *`metadata.name`* is a required value that defines the name of the node
```

Comment:
@tpoitras 
The "metadata.name" is the "full name" of the field, like /usr/bin/ls is the full (absolute) filename of ls.  FWIW, i just added a section describing the full name convention to the [wip doc "Kubernetes Configuration"](http://jenkinscat.gsslab.pnq.redhat.com:8080/job/atomic-branch-tnguyen-kc-yaml/lastSuccessfulBuild/artifact/pub_Kubernetes_Configuration/index.html).